test/JDBC/expected/BABEL-3066-vu-verify.out

```diff
@@ -0,0 +1,131 @@
+SELECT CAST(123456.25 AS real) AS value_as_real, CAST(CAST(123456.25 AS real) AS decimal(30,2)) AS value_casted_to_decimal_30_2;
+GO
+~~START~~
+real#!#numeric
+123456.25#!#123456.25
+~~END~~
+
+
+select cast(cast(123457.145637867 as real) as numeric (30, 20));
```

Comment:
Please make sure all results are verified against SQL server
Add few test case where value is
1) Directly cast to numeric (without real in-between)
2) Use float in-between
3) Value like 0, min real, max real
4) Min and max scale for numeric (0 -> 38)
5) Cast expressions value with diff operators to numeric
6) Expression over casted value -> cast(x) + cast(y)

Reply:
Updated in latest revision.